### PR TITLE
FRDQ-44 - Add getAllActive endpoint for Socials

### PR DESCRIPTION
**Pull request by PiotrGrobelak**

### CODE REVIEW

File: backend/src/controllers/social.controller.ts [hunk 1]
```diff
@@ -16,6 +16,19 @@ export const getAll = async (_req: Request, res: Response) => {
   }
 };
 
+export const getAllActive = async (_req: Request, res: Response) => {
+    try {
+    const socials = await Social.find({active: false});
```
Comment (hayuna) [inline backend/src/controllers/social.controller.ts:21]: Are you sure about this condition? We want to get only active, not inactive ones

Comment (PiotrGrobelak) [reply backend/src/controllers/social.controller.ts:21]: OK sorry. I forgot to switch flag :)

File: backend/src/controllers/social.controller.ts [hunk 2]
```diff
@@ -16,6 +16,19 @@ export const getAll = async (_req: Request, res: Response) => {
   }
 };
 
+export const getAllActive = async (_req: Request, res: Response) => {
+    try {
+    const socials = await Social.find({active: true});
+    if (socials.length === 0) {
+      return res.status(StatusCode.NOT_FOUND).json({ message: 'Deactive Socials not found' });
```
Comment (hayuna) [inline backend/src/controllers/social.controller.ts:23]: - [ ] Please change Incorrect error message 
- [ ] Please add this functionality to unit tests

**Review by hayuna — CHANGES_REQUESTED** [review]
Incorrect error message
Missing unit tests

Rejected

File: backend/src/controllers/social.controller.ts (hunk 2 above)
Comment (PiotrGrobelak) [reply backend/src/controllers/social.controller.ts:23]: OK

**Pull request merged**